token/cli/src/main.rs

```diff
@@ -1577,6 +1609,18 @@ fn main() {
                         .help("Specify the sending token account \
                             [default: owner's associated token account]")
                 )
+                .arg(
+                    Arg::with_name("owner")
+                        .long("owner")
+                        .value_name("KEYPAIR")
+                        .validator(is_valid_signer)
+                        .takes_value(true)
+                        .help(
+                            "Specify the owner of the sending token account. \
+                            This may be a keypair file, the ASK keyword. \
+                            Defaults to the client keypair.",
+                        ),
```

Comment:
Can we put this (and others with duplication) behind a helper function so it will be easier to keep them consistent?  I'm already itching to update the `value_name` and `help` since all signer types are now supported

Reply:
Great call, done!